Modules/EMCAL/src/CellTask.cxx

```diff
@@ -240,16 +301,17 @@ void CellTask::monitorData(o2::framework::ProcessingContext& ctx)
     auto bcphase = trg.mInteractionRecord.bc % 4; // to be fixed:4 histos for EMCAL, 4 histos for DCAL
     auto histos = mHistogramContainer[trgClass];
     std::fill(numCellsSM.begin(), numCellsSM.end(), 0);
+    std::fill(numCellsSM_Thres.begin(), numCellsSM_Thres.end(), 0);
 
     // iterate over subevents
+    int ncell = 0;
```

Comment:
This now also becomes unused, because the information is obtained from the per-supermodule number of cells, but can be cleaned in the next iteration